src/metabase/pulse/render/js_svg.clj

```diff
@@ -69,25 +69,31 @@
       node)))
 
 (defn- parse-svg-string [^String s]
-  (let [s       (str/replace s #"<svg " "<svg xmlns=\"http://www.w3.org/2000/svg\" ")
+  (let [s       (str/replace s #"<svg" "<svg xmlns=\"http://www.w3.org/2000/svg\"")
         factory (SAXSVGDocumentFactory. "org.apache.xerces.parsers.SAXParser")]
     (with-open [is (ByteArrayInputStream. (.getBytes s StandardCharsets/UTF_8))]
       (.createDocument factory "file:///fake.svg" is))))
 
-(def svg-render-width
+(def ^:dynamic ^:private *svg-render-width*
   "Width to render svg images. Intentionally large to improve quality. Consumers should be aware and resize as
   needed. Email should include width tags; slack automatically resizes inline and provides a nice detail view when
   clicked."
   (float 1200))
 
+(def ^:dynamic ^:private *svg-render-height*
+  "Height to render svg images. If not bound, will preserve aspect ratio of original image."
+  nil)
```

Comment:
I don't have a good understand of how the SVG to PNG rendering works, but for some reason the icon doesn't render properly unless the height is set in addition to the width.